### PR TITLE
[service.subtitles.subtitool] 1.1.1

### PR DESCRIPTION
**Pull request by KianMax**
First Version

### CODE REVIEW

File: service.subtitles.subtitool/service.py [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+        ## add it to list, this can be done as many times as needed for all subtitles found
+        xbmcplugin.addDirectoryItem(handle=int(sys.argv[1]), url=url, listitem=listitem, isFolder=False)
+
+
+def Download(language, hash, filename, dlLink):
+    subtitle_list = []
+    ## Cleanup temp dir, we recomend you download/unzip your subs in temp folder and
+    ## pass that to XBMC to copy and activate
+    if xbmcvfs.exists(__temp__):
+        shutil.rmtree(__temp__)
```
Comment (phil65) [inline service.subtitles.subtitool/service.py]: did you try xbmcvfs.rmdir()?
https://romanvm.github.io/Kodistubs/_autosummary/xbmcvfs.html#xbmcvfs.rmdir

**Comment by phil65** [comment]
one comment, rest looks fine. thx.

**Comment by learningit** [comment]
One small comment from me: we're trying to move away from using \__label\__ label format to avoid confusion with Python internals. I see it's used a few times in the above.

File: service.subtitles.subtitool/resources/lib/SubTiTool.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+        try:
+            response = urllib.urlopen(dllink)
+        except urllib2.URLError, e:
+            sys.stderr.write(e.message)
+            return False
+
+        try:
+            srtdata = response.read()
+            open(self.filename, "w").write(srtdata)
```
Comment (phil65) [inline service.subtitles.subtitool/resources/lib/SubTiTool.py]: oh, one more:
you should close the file again after writing, best by using with() statement.

with open(self.filename, "w") as file:
   file.write(srtdata)

**Comment by KianMax** [comment]
Thanks for your comments. Please check my new commit.

**Comment by Razzeee** [comment]
Just as a heads up, please squash these down to one commit, after it's reviewed.

**Comment by learningit** [comment]
Please resubmit as a single commit and I'll merge into the repo. Thanks.

**Comment by phil65** [comment]
For next time:
you dont need to close the PR, just force-push to your branch and the PR will update.

**Pull request closed without merging**